### DISCUSSION TITLE
Ownership Transfer

### DISCUSSION THREAD

**Discussion by BryanLeong99** [codemagic-ci-cd/codemagic-docs#2108] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

Hi there, I would like to transfer my ownership to another member, however, I am using a free plan. Is there a way for me to do so? 

### Expected results

-

### Actual results

-

**Comment by rafeysup93** [comment, marked as answer]
Hi @BryanLeong99, Thanks for reaching out.

You won't be able to do this on a free plan. You can make this change by creating a TEAM.

Perform the following steps:

1. Create a team, if there isn't
2. Add the apps that are necessary
3. Invite the new user's email.
4. Make them as owners of the team.
5. Downgrade the role of the old email user in the team.
6. Remove the old user.
7. Then you can remove the apps from shared applications in the team, which will move the apps to the new users' account, and then delete the team(If not required)

Note: Step 7 is only applicable if you don't want teams!

**Reply by BryanLeong99** [reply]
Thanks for your prompt response. If I would like to create a team, do I need to upgrade my plan or make any payment? I ask this because I was asked to provide my payment details.

**Reply by rafeysup93** [reply]
@BryanLeong99, The team will be invoiced for paid usage every month, commencing on the first day of each month, following the Pay-as-You-Go pricing [structure](https://codemagic.io/pricing/).

**Reply by bakua** [reply]
Hello, after deleting the team, will it keep the new user's account on pay as you go pricing with no free minutes?
